tools/gentool/gentool.go

```diff
@@ -73,25 +114,49 @@ func main() {
 	fieldWithIndexTag := flag.Bool("fieldWithIndexTag", false, "generate field with gorm index tag")
 	fieldWithTypeTag := flag.Bool("fieldWithTypeTag", false, "generate field with gorm column type tag")
 	flag.Parse()
-
-	db, err := connectDB(DBType(*dbType), *dsn)
-	if err != nil {
-		log.Fatalln("connect db server fail:", err)
+	if *genPath != "" {
+		return loadConfigFile(*genPath)
```

Comment:
return config here will lead to lossing information which specified directly in command line arguments